app/models/katello/glue/candlepin/product.rb

```diff
@@ -70,7 +70,7 @@ def orphaned?
       end
 
       def build_product_content(attrs)
-        @productContent = attrs.collect { |pc| Candlepin::ProductContent.new pc }
+        @productContent = attrs.collect { |pc| ::Katello::Candlepin::ProductContent.new pc }
       end
```

Comment:
`::` at the beginning just means use an absolute path to the module vs. a relative path, my thinking was absolute path would better ensure the correct class is loaded. More info here http://stackoverflow.com/questions/10482772/rubys-double-colon-operator-usage-differences